mode/type/standalone/repository/provider/jdbc/mysql/src/main/java/org/apache/shardingsphere/mode/repository/standalone/jdbc/mysql/MysqlJDBCRepositoryProvider.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.mode.repository.standalone.jdbc.mysql;
+
+import org.apache.shardingsphere.mode.repository.standalone.jdbc.provider.JDBCRepositoryProvider;
+
+/**
+ * MySQL JDBC repository provider.
+ */
+public final class MysqlJDBCRepositoryProvider implements JDBCRepositoryProvider {
```

Comment:
Rename MysqlJDBCRepositoryProvider to MySQLJDBCRepositoryProvider

Reply:
- This would violate the checkstyle rule, which requires no more than seven uppercase letters in a word.

Reply:
> * This would violate the checkstyle rule, which requires no more than seven uppercase letters in a word.

I think we need to investigate checkstyle rule and adjustment it, beacase the `MysqlJDBCRepositoryProvider` does not conform to specification

Reply:
- I doubled the allowedAbbreviationLength Property of the AbbreviationAsWordInName Moudle.